pkg/kv/kvserver/replica_proposal.go

```diff
@@ -641,7 +641,9 @@ func addSSTablePreApply(
 	return copied
 }
 
-func (r *Replica) handleReadWriteLocalEvalResult(ctx context.Context, lResult result.LocalResult) {
+func (r *Replica) handleReadWriteLocalEvalResult(
+	ctx context.Context, lResult result.LocalResult, withRaftMu bool,
```

Comment:
`withRaftMu` doesn't really tell me whether this is the caller asking for raftMu to be acquired, or whether they're promising that they already hold it. Consider calling this `raftMuHeld`.

Reply:
Done.